apps/finance/app/src/components/AppLayout.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { AppBar, AppView, Button, ButtonIcon, Viewport, font } from '@aragon/ui'
+import MenuButton from './MenuButton/MenuButton'
+
+const AppLayout = ({
+  children,
+  title,
+  afterTitle,
+  smallViewPadding,
+  onMenuOpen,
+  mainButton,
+}) => {
+  return (
+    <Viewport>
+      {({ below }) => (
+        <AppView
+          padding={below('medium') ? smallViewPadding : 30}
+          appBar={
+            <AppBar>
+              <AppBarContainer
+                style={{ padding: below('medium') ? '0' : '0 30px' }}
+              >
+                <Title>
+                  {below('medium') && <MenuButton onClick={onMenuOpen} />}
+                  <TitleLabel>{title}</TitleLabel>
+                  {afterTitle}
+                </Title>
+                {mainButton &&
+                  (below('medium') ? (
+                    <ButtonIcon
+                      onClick={mainButton.onClick}
+                      title={mainButton.label}
+                      css={`
+                        width: auto;
+                        height: 100%;
+                        padding: 0 20px 0 10px;
```

Comment:
`padding: 0 20px`?

Reply:
It really should be `10px` on both sides, the `20px` on the right side is to reach the right side of the screen, the same way `height: 100%` reaches the top and bottom of the header. This is to make the touch target bigger.